eng/scripts/packagevalidation/PrePublish-Validation.ps1

```diff
@@ -0,0 +1,97 @@
+[CmdletBinding()]
+Param (
+  [ValidateNotNullOrEmpty()]
+  [string] $PackagePath,
+  [string] $ArtifactName,
+  [string] $WorkingDirectory
+)
+
+. "${PSScriptRoot}\..\common\scripts\logging.ps1"
+. "${PSScriptRoot}\..\common\scripts\SemVer.ps1"
+
+$WinVersion = (Get-ChildItem -Path "C:\Program Files (x86)\Windows Kits\10\bin\10.0*" | Sort-Object -Descending)[0]
```

Comment:
This seems to assume windows is there anything that enforces that.

Reply:
I guess we can condition the entire step that runs this script on if its a windows agent

Reply:
We will have to and we will likely need to also be careful and ensure this is the correct path to those tools.